assets/script.js

```diff
@@ -0,0 +1,204 @@
+const resume = require("resume.json")
```

Comment:
I think you got off to a very orderly start (which is more than I can say for myself) so it was only a matter of time before you got this done. If process is the most important aspect, then I think you did a good job of working through the process.

Reply:
Agreed you have all the necessary parts here.